lib/src/util/bolt_helpers.cc

```diff
@@ -14,8 +14,8 @@
 
 namespace fs       = boost::filesystem;
 namespace alg      = boost::algorithm;
-namespace lth_curl = leatherman::curl;
-namespace lth_jc   = leatherman::json_container;
+// namespace lth_curl = leatherman::curl;
```

Comment:
I'm assuming you were testing that we could remove these lines. Looks like the jenkins build worked so we should be 👍 to remove them now.